modules/dataframeqt/src/dataframemodel.cpp

```diff
@@ -200,14 +191,26 @@ QVariant DataFrameModel::data(const QModelIndex& index, int role) const {
             return valueFuncs_[index.column()](index.row());
         case Qt::ToolTipRole:
             return tooltipFuncs_[index.column()](index.row());
-        case Qt::BackgroundRole:
+        case Qt::BackgroundRole: {
+            const auto& indexCol = data_->getIndexColumn()
+                                       ->getTypedBuffer()
+                                       ->getRAMRepresentation()
+                                       ->getDataContainer();
+
+            const bool highlighted =
+                manager_ ? (manager_->isHighlighted(indexCol[index.row()]) ||
+                            manager_->isHighlighted(index.column(), BrushingTarget::Column))
+                         : false;
+            const bool selected = manager_ ? manager_->isSelected(indexCol[index.row()]) : false;
+
             if (highlighted) {
                 return QBrush(QColor(102, 87, 50));
             } else if (selected) {
-                return QBrush(QColor("#47474b"));
+                return QBrush(QColor("#47477b"));
```

Comment:
Change by accident? Blueish purple instead of dark gray?

Reply:
No, the current one has way to little contrast, and is too similar to the hover color. you might select a different color, but is has to be clearly visible...